From 38c329ebac91d446007e781bdcabc663fed25f87 Mon Sep 17 00:00:00 2001
From: Christophe Priouzeau <christophe.priouzeau@st.com>
Date: Wed, 19 Jan 2022 15:17:19 +0100
Subject: [PATCH 1/3] python-lttngust/Makefile.am: Add --install-lib to
 setup.py

Otherwise it may install to /usr/lib, but should be /usr/lib64 when cross
building.

Upstream-Status: Submitted [https://github.com/lttng/lttng-ust/pull/59]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 src/python-lttngust/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python-lttngust/Makefile.am b/src/python-lttngust/Makefile.am
index f95482d..f76d95b 100644
--- a/src/python-lttngust/Makefile.am
+++ b/src/python-lttngust/Makefile.am
@@ -32,7 +32,7 @@ install-exec-local: build-python-bindings.stamp
 	if [ "$(DESTDIR)" != "" ]; then \
 		opts="$$opts --root=$(DESTDIR)"; \
 	fi; \
-	$(PYTHON) $(builddir)/setup.py install $$opts;
+	$(PYTHON) $(builddir)/setup.py install $$opts --install-lib=$(pythondir);
 
 clean-local:
 	rm -rf $(builddir)/build
-- 
2.17.1

